src/components/screens/LoginScreen.tsx

```diff
@@ -1,26 +1,97 @@
+import { Formik, FormikProps } from 'formik';
 import React from 'react';
-import { Button, SafeAreaView, Text } from 'react-native';
+import { SafeAreaView, StyleSheet, View } from 'react-native';
 import { ConnectedProps, connect } from 'react-redux';
 
 import { login } from '../../redux/modules/auth';
+import { FormErrors, FormField, WobblyText } from '../atoms';
+import WobblyButton, { Intent } from '../atoms/WobblyButton';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  welcomeHeading: {
+    textAlign: 'center',
+    marginBottom: 10,
+  },
+});
 
 const mapDispatch = {
   login,
 };
 const connector = connect(undefined, mapDispatch);
 type PropsFromRedux = ConnectedProps<typeof connector>;
 
-const LoginScreen: React.FC<PropsFromRedux> = ({ login }) => {
-  const loginFlow = () => {
-    login('dev@xmpp.wobbly.app', 'Cee9ech4Ia6wupho');
-  };
-
-  return (
-    <SafeAreaView>
-      <Text>Please login!</Text>
-      <Button onPress={loginFlow} title="Login" />
-    </SafeAreaView>
-  );
-};
+export interface LoginFormFields {
+  email: string;
+  password: string;
+}
+
+class LoginScreen extends React.PureComponent<PropsFromRedux> {
+  constructor(props: any) {
+    super(props);
+    this.loginFlow = this.loginFlow.bind(this);
+  }
+
+  loginFlow(values: any) {
+    const { email, password } = values;
```

Comment:
this line is going to throw linter warnings until we actually use it -- i suggest putting the login credentials of our demo account into the form's `initialValues`. that'll also make it slightly easier to set up proper auth later!

Reply:
It turns out Formik does this implicitly, so I'm just going to remove the passing of values for now.